repo/packages/M/marathon/0/config.json

```diff
@@ -0,0 +1,378 @@
+{
+  "additionalProperties" : false,
+  "description" : "Marathon universe properties",
+  "properties" : {
+    "marathon" : {
+      "additionalProperties" : false,
+      "description" : "Marathon App Definition properties",
+      "properties" : {
+        "id" : {
+          "description" : "The id of this marathon application name.",
+          "type" : "string"
+        },
+        "cpus" : {
+          "default" : 2,
+          "description" : "CPU shares to allocate to each Marathon instance.",
+          "minimum" : 0,
+          "type" : "number"
+        },
+        "mem" : {
+          "default" : 1024,
+          "description" : "Memory (MB) to allocate to each Marathon task.",
+          "minimum" : 512,
+          "type" : "number"
+        },
+        "instances" : {
+          "default" : 1,
+          "description" : "Number of Marathon instances to run.",
+          "minimum" : 0,
+          "type" : "integer"
+        },
+        "uris" : {
+          "default" : [ ],
+          "description" : "List of URIs that will be download and made available in the current working directory of Marathon. For example this can be used to download a Java keystore file for SSL configuration.",
+          "items" : {
+            "pattern" : "^[\\s]+",
+            "type" : "string"
+          },
+          "type" : "array"
+        }
+      },
+      "required" : [ "cpus", "mem", "instances" ],
+      "type" : "object"
+    },
+    "jvm" : {
+      "additionalProperties" : false,
+      "description" : "JVM specific configuration properties",
+      "properties" : {
+        "heap-min" : {
+          "default" : 256,
+          "description" : "Memory (MB) start size for the JVM heap. This number should be be less or equals than the jvm-heap-max.",
+          "minimum" : 0,
+          "type" : "integer"
+        },
+        "heap-max" : {
+          "default" : 768,
+          "description" : "Memory (MB) max size for the JVM heap. This number should be be less than the absolute memory.",
+          "minimum" : 0,
+          "type" : "integer"
+        }
+      },
+      "required" : [ "heap-min", "heap-max" ],
+      "type" : "object"
+    },
+    "commandLine" : {
+      "additionalProperties" : true,
+      "description" : "Marathon command line properties",
```

Comment:
Since we call it command line flags elsewhere, I also would use the name here.

I think we should probably mention that these directly map to the flags described here: https://mesosphere.github.io/marathon/docs/command-line-flags.html